packages/design-system/src/styles/components/_VerticalNav.scss

```diff
@@ -75,6 +83,24 @@
     @extend %nav-item-focus;
   }
 
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    /* stylelint-disable order/properties-alphabetical-order */
```

Comment:
I have to add this here to get `-ms-high-contrast-adjust: none;` to cascade properly on lines 89 and 94.